DMan/HW6/6.1.py

```diff
@@ -0,0 +1,30 @@
+            #EVEN NUMBERS WHICH ARE DEVISEBLE BY 2
+
```

Comment:
good job
look at this code
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')

Reply:
conflict was resolved